README.md

```diff
@@ -120,37 +120,41 @@ controller_server:
       time_steps: 15
       model_dt: 0.1
       batch_size: 400
-      vx_std: 0.1
-      vy_std: 0.1
-      wz_std: 0.6
+      vx_std: 0.3
+      vy_std: 0.3
+      wz_std: 1.3
       vx_max: 0.5
       vy_max: 0.5
       wz_max: 1.3
       iteration_count: 2
       temperature: 0.25
       motion_model: "DiffDrive"
       visualize: false
-      critics: [ "GoalCritic", "PreferForwardCritic", GoalAngleCritic", "PathAngleCritic", "ReferenceTrajectoryCritic", "ObstaclesCritic" ]
+      critics: [ "GoalCritic", "GoalAngleCritic", "ObstaclesCritic", "ReferenceTrajectoryCritic"]
```

Comment:
No prefer forward / path angle?

Reply:
Not adding twirling makes sense

Reply:
Maybe this is why it tracks the path backwards sometimes (either or, or both?)

Reply:
i'll add this. I'll noticed that goal critic / path_angle critic lead sometimes to local minimum. I reworked reference critic to consider "local" goals, that's greatly improves behavior. I planning to make the same behavior for angles